en/developer_guide/logs.rst

```diff
@@ -0,0 +1,89 @@
+Logs
+====
+
+For logs we use Monolog. The Logs table is displayed in administration section in the backoffice.
+It displays the date and time, user's IP, user's name, the current site's name and the log's message.
```

Comment:
Our backoffice logging is based on Monolog. We provide an interface to display all the user action on the website